libs/ledger-live-common/src/families/solana/api/chain/web3.ts

```diff
@@ -260,6 +260,17 @@ export function getStakeAccountMinimumBalanceForRentExemption(api: ChainAPI) {
   return api.getMinimumBalanceForRentExemption(StakeProgram.space);
 }
 
+export async function getAccountMinimumBalanceForRentExemption(
+  api: ChainAPI,
+  address: string
+) {
+  const accInfo = await api.getAccountInfo(address);
+  const accSpace =
+    accInfo !== null && "parsed" in accInfo.data ? accInfo.data.space : 0;
```

Comment:
Hi @haammar-ledger , thanks for review. I think in this case if `accInfo` is not null then the type system guarantees that `data` is also not null since it's a non-optional field of `accInfo`. Thus checking that `data` is defined is reduntant.

Reply:
Hi @konoart, just for the follow-up, because it happens that we've been bitten by this just today in a different context!

Any data that's coming from backends is typed at runtime and therefore isn't protected by static typing, see https://www.typescriptlang.org/play?#code/IYZwngdgxgBAZgV2gFwJYHsIwOYFNkAiwywAFAJQBcMACgE7oC2qIuAPMBGAHwwDeAKBjCYdfAjpZBIkQCIQTXHFS4ANgBNZ1WQEZZAGiEiAvgG4BxgQNCRYiFBiyNgqCBX5GomEMhjriwDAAvDDAAO4uvniEARTmwl4QPjAKjEoqGtQQCIwARrh0wX4BAHSp6Wrq8TCJCqq4Jaro2KTIYAAOuOhwKYrKleSmMAD0wzDyyHSu2LIWVs6ucUA

Just FYI because we talked about it, but really no big deal here. :slightly_smiling_face: